src/MICore/Transports/StreamTransport.cs

```diff
@@ -147,6 +147,12 @@ protected void Echo(string cmd)
             lock (_locker)
             {
                 _writer?.WriteLine(cmd);
+                // https://github.com/Microsoft/MIEngine/issues/616 : If it is local gdb (MinGW/Cygwin) on Windows, we need to send an extra line after commands 
+                // so that if it errors, the error will come through. 
+                if((_callback is Debugger) && ((Debugger)_callback).IsLocalGdbOnWindows)
```

Comment:
Checking if '_callback' is Debugger feels kind of fragile. Could we instead fix it in Debugger.cs line 1450 with 'if (IsLocalGdbOnWindows) _transport.Send(string.Empty);